internal/godplugin/job/job.go

```diff
@@ -7,20 +7,21 @@ import (
 	"github.com/l2isbad/go.d.plugin/internal/modules"
 )
 
-func New(m modules.Module, c *Config) *Job {
+func New(m modules.Module, c Config) *Job {
 	_, u := m.(modules.Unsafer)
 
 	return &Job{
 		Module: m,
-		Config: c,
+		C:      c,
 		unsafe: u,
 	}
 }
 
 type Job struct {
 	modules.Module
-	*Config
-	timers
+	t       timers
+	C       Config
+	W       *WrappedCharts
```

Comment:
I think full name is more readable.
```
timers       timers
Config       Config
Charts       *WrappedCharts
```

Reply:
Ofc it's more readable. `timers` and `Config` should be embedded.